crates/bevy_time/src/fixed_timestep.rs

```diff
@@ -155,7 +169,12 @@ impl Default for LocalFixedTimestepState {
 impl LocalFixedTimestepState {
     fn update(&mut self, time: &Time) -> ShouldRun {
         if !self.looping {
-            self.accumulator += time.delta_seconds_f64();
+            let time_delta = time.delta_seconds_f64();
+            self.accumulator += if time_delta > self.max_time_delta {
+                self.max_time_delta
+            } else {
+                time_delta
+            };
```

Comment:
That wouldn't achieve the desired goal, because the system would run the maximum number of steps (up to the limit) until it has caught up to the accumulator.

Reply:
I meant that we should enable limiting the number of steps per frame as a supplement to pausing `Time`.

I think it's important that the fixed timestep be kept in sync with `Time`. Your change would do the opposite.